ambry-store/src/main/java/com.github.ambry.store/IndexSegment.java

```diff
@@ -266,6 +266,7 @@ IndexValue find(StoreKey keyToFind) throws StoreException {
         return index.get(keyToFind);
       } else {
         // check bloom filter first
+        metrics.bloomAccessedCount.inc();
```

Comment:
Not sure when `bloomFilter == null` scenario occurs, but let us separate that out when updating the metrics since that is not really "accessing" bloom filter. Plus, from the code, that seems like the only thing other than true false positives that can result in an unsuccessful search of the segment.